src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,266 @@
+# Imaging data types
```

Comment:
What does imaging mean to you? The sense in which I'm using it is "at least one dimension (and usually two or three) have a spatial interpretation". I don't know that these things apply to N-D data generally.

Can you clarify your objections?

Reply:
The term "N-Dimensional digital data" also applies to raw EEG (space and time), to surround audio (space and time), to photo (2D space), video (2D pixel space and time), eye tracker (which eye, pupil versus gaze, and time), polhemus electrode positions (space), motion capture (which body part, and time), touch screen events in behavioral data (space and time). The subsequent text does not provide  guidelines how to deal with those data types. It does not even specify how the temporal dimension of MRI is dealt with, only deals with the different spatial representations of MRI data (volumetric, surface-based, parcellated). 

Surface-based and parcellated descriptions of data are by themselves not images any more, but are derived from imaging data. That is why I consider "Imaging data types"  an appropriate title. Or perhaps "Data types derived from images". 

Interestingly, "Data types derived from images" would also apply (in case BEP021 moves further ahead) to volume conduction models of the head and source models, used in EEG and MEG pipelines, and the functional estimates that are obtained for those source models (similar to how we store HCP MEG results in CIFTI). These results are multi-modal, since the input is anatomical MRI and electrophys.

Reply:
@effigies sorry for the lack in clarity. What I mean is that traversing over the whole taxonomy of the dot product of data types and modalities seems impractical.

I agree common-derivatives needs to be extra-careful not to impose some particular modality idiosyncrasy to the rest of them. To address that, the original BIDS spec only splits by modalities in a very broad sense and then BEPs followed that approach. I just can't see why this extra granularity helps to get common-derivatives merged faster or helps to better describe derivatives. IMHO it will be much harder to keep the consistency across BEPs, and also, this increases communication and review costs at no clear benefit.

Reply:
The derivatives should also allow the representation and description of processed `physio` data, which can be (on its own) in the `beh` folder according to the current spec. Physio can for example be GSR, ECG or optical heart signals, in relation to events. So rather than `N>1` it should be `N=>1`. 

And how about processed behavioral data represented in the (non-continuous) events? Is it appropriate to describe irregularly sampled data as N-D?

Reply:
> The derivatives should also allow the representation and description of processed `physio` data, which can be (on its own) in the `beh` folder according to the current spec. Physio can for example be GSR, ECG or optical heart signals, in relation to events. So rather than `N>1` it should be `N=>1`.
> 
> And how about processed behavioral data represented in the (non-continuous) events? Is it appropriate to describe irregularly sampled data as N-D?

Sorry, I can't understand what you are suggesting here. You would not agree with _Imaging derivates_ as the main header for this split?